cost/aws/superseded_instances/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.2
+
+- updated version to match meta parent and child policy
```

Comment:
This isn't what should be in the CHANGELOG. That was a change you made to resolve a merge conflict, not the actual pertinent change to the policy that a user would be concerned about.

This should say something about the README link being corrected, since that is the actual substantive change.

Reply:
Correct, let me change that